pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -25,6 +27,15 @@ import (
 
 var log = logf.Log.WithName("controller_baremetalhost")
 
+// DeprovisionRequeueDelay controls the amount of time the controller
+// waits between attempts to determine if the deprovisioning operation
+// has been completed.
+//
+// FIXME(dhellmann): This is public for now so we can change it in the
+// test suite. When we figure out a better unit test setup we can make
+// it private.
+var DeprovisionRequeueDelay = time.Second * 10
```

Comment:
I think this would be a reasonable attribute of `ReconcileBareMetalHost`. Feel free to do that now or later as you prefer.

Reply:
good call

Reply:
I'm not sure how I'd be able to get to the reconciler being used by the current tests until after the conversion to unit tests, so I'll hold off on that